### PR TITLE
[enterprise-4.12] OSDOCS-4053: RN entry

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.12

Issue: https://issues.redhat.com/browse/OSDOCS-4053

Link to docs preview: http://file.rdu.redhat.com/jdohmann/OSDOCS-3888-RN/release_notes/ocp-4-12-release-notes.html#ingress-controller-autoscaling

Additional information: RN entry for #49786

### CODE REVIEW

**Comment by jldohmann** [comment]
@ShudiLi could you PTAL for QE ack? This is the RN entry for [NE-361](https://issues.redhat.com/browse/NE-361). Thank you

**Comment by ShudiLi** [comment]
/label qe-approved
@jldohmann thanks

**Comment by mburke5678** [comment]
@jldohmann LGTM

**Comment by jldohmann** [comment]
b9c794f..1317697 add ID

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -74,6 +74,13 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-networking"]
 === Networking
 
+[id="ocp-4-12-nw-ingress-autoscaling"]
+==== Ingress Controller Autoscaling
+
+You can now use the {product-title} Custom Metrics Autoscaler operator to dynamically scale the default Ingress Controller based on metrics in your deployed cluster, such as the number of worker nodes available.
```
Comment (kalexand-rh) [inline release_notes/ocp-4-12-release-notes.adoc:80]:
```suggestion
You can now use the {product-title} Custom Metrics Autoscaler Operator to dynamically scale the default Ingress Controller based on metrics in your deployed cluster, such as the number of worker nodes available.
```

Comment (jldohmann) [reply release_notes/ocp-4-12-release-notes.adoc]: thanks Kathryn. fixed

**Pull request merged**